async_work_thread_safe_function/napi/binding.c

```diff
@@ -85,26 +91,31 @@ static void ExecuteWork(napi_env env, void* data) {
 
       // Initiate the call into JavaScript. The call into JavaScript will not
       // have happened when this function returns, but it will be queued.
-      assert(napi_call_threadsafe_function(addon_data->tsfn,
+      status = napi_call_threadsafe_function(addon_data->tsfn,
                                            the_prime,
-                                           napi_tsfn_blocking) == napi_ok);
+                                           napi_tsfn_blocking);
+      assert(status == napi_ok);
     }
   }
 
   // Indicate that this thread will make no further use of the thread-safe function.
-  assert(napi_release_threadsafe_function(addon_data->tsfn,
-                                          napi_tsfn_release) == napi_ok);
+  status = napi_release_threadsafe_function(addon_data->tsfn,
+                                          napi_tsfn_release);
+  assert(status == napi_ok);
 }
 
 // This function runs on the main thread after `ExecuteWork` exits.
 static void WorkComplete(napi_env env, napi_status status, void* data) {
   AddonData* addon_data = (AddonData*)data;
+  napi_status s;
```

Comment:
A nit but I'd prefer a more complete name, for example `status` versus the one letter name